site/content/en/docs/Examples/custom-controller.md

```diff
@@ -0,0 +1,109 @@
+---
+title: "Custom Controller for Agones Game Servers"
+linkTitle: "Custom Controller"
+date:
+publishDate:
+description: >
+  This Custom Controller example shows how to create, deploy and run a Custom Kubernetes Controller for Agones that logs changes to GameServers and modifies their labels.
+---
+
+## Prerequisite
+
+ To get started, ensure the following prerequisites are met:
+
+  - You have a running Kubernetes cluster.
+  
+  - Agones is installed on your cluster. See [Agones guide](https://agones.dev/site/docs/installation/install-agones/).
+
+  - Example custom controller code is available {{< ghlink href="examples/custom-controller" >}}here{{< /ghlink >}}
+
+## Deploy the Custom Controller
+
+For a quick deployment of the custom controller on your cluster, execute:
+
+```bash
+kubectl apply -f deployment.yaml
+```
+
+When you run this command, it quickly sets up your controller by doing four things: 
+ - Creating a service account for secure communication with Kubernetes
+ - Defining a role with the right permissions to handle game servers and events
+ - Linking this role to the account for broad access
+ - Launching two controllers for reliability.
+
+## Verify the Controller
+
+To ensure the custom controller is operational, execute the following command. You should see two instances of the controller actively running:
+
+```bash
+kubectl get pods -n agones-system
+```
+
+## Monitor the log events for the custom controller pod
+
+To monitor the log events of the custom controller pod during the creation, modification, and deletion of game servers, use the following command:
+
+```bash
+kubectl logs -f <custom-controller-pod> -n agones-system
+```
+
+**Note**: If a custom controller runs into trouble with logging events, the backup controller will automatically assume the leadership role, ensuring uninterrupted logging of event details.
+
+## Deploy the Agones Fleet
+
+To apply the fleet configuration to your cluster, use the following command:
```

Comment:
Can lift the create a fleet markdown from https://agones.dev/site/docs/getting-started/create-fleet/#1-create-a-fleet 

Then they don't need to worry about cloning the whole repository.

Reply:
Currently, I've copied the information from the `create a fleet` section. Would it be sufficient to include the link https://agones.dev/site/docs/getting-started/create-fleet/#1-create-a-fleet is enough?

something like:
```
## Create a Fleet

Please refer to the documentation on creating a fleet available at [this link](https://agones.dev/site/docs/getting-started/create-fleet/#1-create-a-fleet) for detailed information
```